### PR TITLE
Added platform as a category in client apps templates

### PR DESCRIPTION
**Pull request by emilioicai**
Review? @wtrocki @pmdarrow

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -306,7 +306,7 @@
         "githubUrl": "https://github.com/feedhenry/appforms-project-client.git",
         "repoBranch": "refs/heads/master",
         "forcedSelection": true,
-        "category": "Forms",
+        "category": "Cordova",
```
Comment (wtrocki) [inline global.json]: Despite mentioning this in the ticket, I would not list this as Cordova. 
This template is created for Drag and Drop apps. It's not meant to be modified by customers (in most of the cases) and specifying this as Cordova may be misleading.

**Comment by wtrocki** [comment]
@emilioicai - See that there are still projects that are "client based" that have some invalid categories. See `Sample Apps` category. That would resonate with already introduced categories.
How about moving IOS/Android etc. across the board. We can use "multiplatform" for mixed projects? Do we have tickets for that? as this is just cosmetic change we can do this in one PR.

File: global.json (hunk 1 above)
Comment (emilioicai) [inline global.json]: got it, will update the PR

**Comment by emilioicai** [comment]
@wtrocki I'm not sure if those should fall in the scope of this issue since the main reason for it was to clean up the Client App's section. All the `Sample App` templates are part of the Project's apps templates (where filtering by app type is not possible) or Cloud App's templates (where platform is maybe not the right way of categorizing). Shall we merge this one and discuss about how to categorize the rest?

PS: no idea if categorizing is a word but it sounds good in my head :)

**Comment by wtrocki** [comment]
@emilioicai Feel free to merge. I was just raising my concern, but in light of your comment I think that we should stick with current state - at least for now.

**Comment by emilioicai** [comment]
@wtrocki I understand your concern and I would suggest to capture in an issue so we don't lose track of it and discuss properly

**Comment by emilioicai** [comment]
retest this please

**Pull request merged**